pkg/detectors/dockerhub_v2/dockerhub_v2.go

```diff
@@ -0,0 +1,97 @@
+package dockerhub_v2
```

Comment:
It looks like the other versioned detectors don't use underscores in their package names. Can you do the same here for consistency? (i.e. `dockerhubv2`)

Reply:
it says to do so here - add _v2 to package name - https://github.com/trufflesecurity/trufflehog/blob/main/hack/docs/Adding_Detectors_external.md

Reply:
That is an excellent point. I'll make a note to get that cleaned up, thanks!

Reply:
Hot take: underscores make package names easier to read.